package-lock.json

```diff
@@ -1,8 +1,9898 @@
 {
   "name": "npm-upgrade",
   "version": "3.0.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

Comment:
Is it compatible with `npm < 7`?

Reply:
According to the [documentation](https://docs.npmjs.com/cli/v7/configuring-npm/package-lock-json#lockfileversion): yes, it is.

But if you want, I can install an old version and commit it with lockfile-version 1.

Reply:
No, that's fine